app/src/main/java/nodomain/freeyourgadget/gadgetbridge/adapter/GBDeviceAdapter.java

```diff
@@ -59,6 +60,11 @@ public View getView(int position, View view, ViewGroup parent) {
         short batteryLevel = device.getBatteryLevel();
         if (batteryLevel != GBDevice.BATTERY_UNKNOWN) {
             batteryStatusLabel.setText("BAT: " + device.getBatteryLevel() + "%");
```

Comment:
This should be translateable, I guess, but it wasn't before, so we can do it later as well.